### PR TITLE
add helixbot user and give him right to sudo

### PR DESCRIPTION
**Pull request by wfurt**
with this, System.Diagnostics.Process tests can pass.

```
  Finished:    System.Diagnostics.Process.Tests
=== TEST EXECUTION SUMMARY ===
   System.Diagnostics.Process.Tests  Total: 272, Errors: 0, Failed: 0, Skipped: 4, Time: 8.589s
```

the problem is that running tests as root is unusual and there are asserts sprinkled through tests. It is assumed that sudo is way how to gain privileged access.

### CODE REVIEW

File: src/alpine/3.9/helix/amd64/Dockerfile [hunk 1]
```diff
@@ -54,3 +54,9 @@ RUN python -m pip install --upgrade pip==19.0.2 && \
                   requests==2.21.0 \
                   six==1.12.0 \
                   virtualenv==16.2.0
+
+# Fixup helixbot user
+RUN /usr/sbin/adduser -D -G adm -s /bin/sh helixbot; \
```
Comment (MattGal) [inline src/alpine/3.9/helix/amd64/Dockerfile]: I think this needs to be useradd not adduser, trying it out locally...

**Comment by wfurt** [comment]
it should work:
```
bash-4.4# adduser
BusyBox v1.29.3 (2019-01-24 07:45:07 UTC) multi-call binary.
```
I took it from old alpine setup code.

File: src/alpine/3.9/helix/amd64/Dockerfile (hunk 1 above)
Comment (MattGal) [reply src/alpine/3.9/helix/amd64/Dockerfile]: huh nope this did what I'd expect it to.

**Comment by MichaelSimons** [comment]
@dotnet-bot test alpine please.

**Comment by MichaelSimons** [comment]
The following image was built as a result of merging the PR

mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.9-helix-34f2903-20190227035445

**Pull request merged**